http/src/request/channel/reaction/mod.rs

```diff
@@ -10,3 +10,31 @@ pub use self::{
     delete_all_reactions::DeleteAllReactions, delete_reaction::DeleteReaction,
     get_reactions::GetReactions,
 };
+
+use std::fmt::Write;
+use twilight_model::channel::ReactionType;
+
+fn format_emoji(emoji: ReactionType) -> String {
+    match emoji {
+        ReactionType::Custom { animated, id, name } => {
+            let mut emoji = String::from("<");
+
+            if animated {
+                emoji.push('a');
+            }
+
+            emoji.push(':');
+
+            if let Some(name) = name {
+                emoji.push_str(name.as_ref());
+            }
+
+            emoji.push(':');
+            let _ = write!(emoji, "{}", id);
+            emoji.push('>');
+
+            emoji
+        }
+        ReactionType::Unicode { name } => name,
+    }
+}
```

Comment:
As discussed in the Discord the format here should be `name:id` or just `:id`if name is not available.